From ebb6e3b2c547ce9fb2cb50f31313e5e087e6b41a Mon Sep 17 00:00:00 2001
From: Christopher Degawa <ccom@randomderp.com>
Date: Sat, 13 Mar 2021 16:38:05 -0600
Subject: [PATCH 10/10] modules/access_output: don't put -lgpg-error for
 livehttp

it's already included in GCRYPT_LIBS if you use pkg-config

Signed-off-by: Christopher Degawa <ccom@randomderp.com>
---
 modules/access_output/Makefile.am | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/modules/access_output/Makefile.am b/modules/access_output/Makefile.am
index 5a6e82d80f..b39ea8ca7c 100644
--- a/modules/access_output/Makefile.am
+++ b/modules/access_output/Makefile.am
@@ -14,7 +14,7 @@ access_out_LTLIBRARIES = \
 
 libaccess_output_livehttp_plugin_la_SOURCES = access_output/livehttp.c
 libaccess_output_livehttp_plugin_la_CFLAGS = $(AM_CFLAGS) $(GCRYPT_CFLAGS)
-libaccess_output_livehttp_plugin_la_LIBADD = $(GCRYPT_LIBS) -lgpg-error $(LIBM)
+libaccess_output_livehttp_plugin_la_LIBADD = $(GCRYPT_LIBS) $(LIBM)
 if HAVE_GCRYPT
 access_out_LTLIBRARIES += libaccess_output_livehttp_plugin.la
 endif
-- 
2.31.1

